_includes/menu.html

```diff
@@ -7,8 +7,8 @@
         <span class="icon-bar"></span>
         <span class="icon-bar"></span>
       </button>
-      <a class="navbar-brand" href="../">
-        <span>Intercity.io</span>
+      <a class="navbar-brand" href="/">
+        <img src="/images/logo.svg">
```

Comment:
Yes, I think the font from the old logo doesn't match the new UI style from the landing page and Bulma. I could add "Intercity" in the landing page's font however.